central/vulnerabilityrequest/utils/util.go

```diff
@@ -123,12 +124,21 @@ func GetImageQueryForVulnReq(request *storage.VulnerabilityRequest) (*v1.Query,
 	}
 
 	if imageScope := requestScope.GetImageScope(); imageScope != nil {
+		if features.UnifiedCVEDeferral.Enabled() {
+			// Check if it is the v2 way of global scoping.
+			if imageScope.GetRegistry() == common.MatchAll &&
```

Comment:
nit:  Could you make a func to do this so all these if statements would be like 
if v2.GlobalScope(imageScope) {
}

Reply:
great idea